grafeas/synth.py

```diff
@@ -211,8 +185,15 @@ def escape_braces(match):
     'https://googleapis.github.io/google-cloud-ruby'
 )
 
+# https://github.com/googleapis/gapic-generator/issues/2393
+s.replace(
+    'grafeas.gemspec',
+    'gem.add_development_dependency "rubocop".*$',
+    'gem.add_development_dependency "google-style", "~> 1.24.0"'
+)
```

Comment:
This can be removed once the `ruby.update_gemspec` line above is fixed.

Reply:
Should be fixed.